src/main/java/seedu/address/logic/commands/staff/EditStaffDetailsCommand.java

```diff
@@ -42,6 +42,7 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Staff details: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This staff member has already been registered.";
+    public static final String MESSAGE_DOCTOR_ON_DUTY = "You cannot edit while the doctor is on duty";
```

Comment:
Please remove the "You" in the error messages

Reply:
Fixed